website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -638,6 +638,42 @@ While it’s technically possible to use the multi-select feature in a table imp
 
 !!!
 
+### Sorting by selected
+
+To enable sorting by selected rows in a table, you need to set `@selectableColumnKey` to the key in each row that tracks its selection state. This allows you to sort the table based on whether rows are selected or not.
+
+In the demo below, we set up a multi-select table that can be sorted based on the selection state of its rows.
+
+```handlebars
+<Hds::Table
+  @isSelectable={{true}}
+  @selectableColumnKey="isSelected"
+  @onSelectionChange={{this.demoOnSelectionChangeSortBySelected}}
+  @model={{this.demoSortBySelectedData}}
+  @columns={{array
+    (hash key="artist" label="Artist" isSortable=true)
+    (hash key="album" label="Album" isSortable=true)
+    (hash key="year" label="Year" isSortable=true)
+    (hash key="selection" label="Selected" isSortable=true)
+  }}
+  @sortBy="selection"
+  @sortOrder="desc"
+>
+  <:body as |B|>
+    <B.Tr
+      @selectionKey={{B.data.id}}
+      @isSelected={{B.data.isSelected}}
+      @selectionAriaLabelSuffix="row {{B.data.artist}} / {{B.data.album}}"
+    >
+      <B.Td>{{B.data.artist}}</B.Td>
+      <B.Td>{{B.data.album}}</B.Td>
+      <B.Td>{{B.data.year}}</B.Td>
+      <B.Td>{{if B.data.isSelected "Yes" "No"}}</B.Td>
+    </B.Tr>
+  </:body>
+</Hds::Table>
+```
+
```

Comment:
[thought] considering that consumers may have implemented the table without using the `@model/@columns` arguments (second demo in the previous PR) should we add here at least an "info" banner explaining that in that case they should contact us for help on how to make it work? (we're not documenting the props that they should use, so how would they know?
The alternative would be to have a second example here, following a similar implementation as the "demo 6" in the showcase
No strong feelings on which one is better, but I think we need to make the consumers aware of this.

Reply:
That's a good point. I'll document the other approach here.